SDCD-854: Update "Synchronize Repository Metadata to DataDog" section of DORA setup docs to reference the updated github integration tile design (#27948)
DataDog/documentation: content/en/dora_metrics/setup/deployments.md

1. Select at least **Read** repository permissions for **Contents** and **Pull Requests**.
2. Subscribe at least to **Push**, **PullRequest** and **PullRequestReview** events.

> To confirm that the setup is valid, select your GitHub application in the [GitHub integration tile][2] and verify that, in the **Datadog Features** table, the **Pull Request Information** feature is marked as valid.

Reviewer: Optional suggestion

```suggestion
To confirm that the setup is valid, select your GitHub application in the [GitHub integration tile][2] and verify that the **Datadog Features** table shows **Pull Request Information** meets all requirements.
```
